pkg/ui/src/views/databases/containers/databaseGrants/index.tsx

```diff
@@ -27,53 +30,60 @@ const grantsSortSetting = new LocalSetting<AdminUIState, SortSetting>(
 
 // DatabaseSummaryGrants displays a summary section describing the grants
 // active on a single database.
-class DatabaseSummaryGrants extends DatabaseSummaryBase {
+export class DatabaseSummaryGrants extends DatabaseSummaryBase {
   totalUsers() {
     const grants = this.props.grants;
     return grants && grants.length;
   }
 
+  noDatabaseResults = () => (
+    <>
+      <h3 className="table__no-results--title"><DatabaseIcon />This grands has no users.</h3>
```

Comment:
Can you check designs or with Anne/Ryan about this text? It doesn't make sense as is to me.

Maybe something like "No grants on this table".

Reply:
done